common/subdivisions/en.xml

```diff
@@ -1650,6 +1650,7 @@ Generated by tool:	SubdivisionNode
 			<subdivision type="gbeln">East Lothian</subdivision> <!-- in gbsct : Scotland -->
 			<subdivision type="gbels">Outer Hebrides</subdivision> <!-- in gbsct : Scotland -->
 			<subdivision type="gbenf">Enfield</subdivision> <!-- in gbeng : England -->
+			<subdivision type="gbeng">England</subdivision> <!-- deprecated -->
```

Comment:
Hmm. Does the deprecation of gbeng, gbsct, gbwls have any impact on the use of those in emoji tag sequences? I thought those definitions depended on CLDR validity...